dts/common/common.dts

```diff
@@ -2,7 +2,3 @@
  * Common Device Tree source, used for conditionally pulling in features and
```

Comment:
It is no longer possible to conditionally pull in features. So we shouldn't have a file saying
that we can do so.

Reply:
Fixed.